stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -63,11 +64,14 @@
     private static final String LOGGING_ENDPOINT = "https://m.stripe.com/4";
 
     private static final String CHARSET = "UTF-8";
+    private static final String CUSTOMERS = "customers";
     private static final String TOKENS = "tokens";
     private static final String SOURCES = "sources";
     private static final String DNS_CACHE_TTL_PROPERTY_NAME = "networkaddress.cache.ttl";
     private static final SSLSocketFactory SSL_SOCKET_FACTORY = new StripeSSLSocketFactory();
 
+    static final String API_VERSION = "2015-10-12";
```

Comment:
maybe we can just start with the latest API version. we have this version in the iOS SDK because it was already there before we added ephemeral key support and weren't sure of the implications of changing it.